server/emailAPI/IEmailAPI.ts

```diff
@@ -0,0 +1,14 @@
+import IEmailVerificationTemplate from "../serverAPI/model/emailVerification/IEmailVerificationTemplate";
+
+export default interface IEmailAPI {
+    /**
+     * Sends verification code to the email address.
+     * 
+     * @param to list of recievers' email addresses.
```

Comment:
You say here that `to` is a list, but the parameter below is a string. which should it be?

Reply:
Also, I guess it's late for this comment, but see the [style guide](https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments) for format for commments; their approach makes it easy to distinguish the parameter name from the description.